src/components/Comments/ReplyCommentItem.js

```diff
@@ -222,7 +303,7 @@ const ReplyCommentItem = ({
             )}
           </ButtonHightlight>
         </TouchableOpacity>
-        <Text style={styles.vote(voteStyle())}>{totalVote}</Text>
+        {totalVote !== 0 && <Text style={styles.vote(voteStyle())}>{totalVote}</Text>}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition `totalVote !== 0` is added to the rendering of the total vote count. This will hide the vote count when it's zero. If this is the intended behavior, then it's fine. However, if you want to display '0' when there are no votes, you should remove this condition.

```diff
- {totalVote !== 0 && <Text style={styles.vote(voteStyle())}>{totalVote}</Text>}
+ <Text style={styles.vote(voteStyle())}>{totalVote}</Text>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->